tests/conftest.py

```diff
@@ -0,0 +1,107 @@
+import datetime
+
+from flask import template_rendered
+from konsent import app
+
+from unittest.mock import MagicMock
```

Comment:
Throws `ImportError: No module named mock`

Reply:
again, this is a python3 module and it was there since i wrote the first unittest. in python2 this doesn't exist